bridge/device/cloud/signIn.go

```diff
@@ -71,8 +71,12 @@ func (c *Manager) signIn(ctx context.Context) error {
 		return err
 	}
 	if resp.Code() != codes.Changed {
-		if resp.Code() == codes.Unauthorized && creds.RefreshToken == "" {
-			c.cleanup()
+		if resp.Code() == codes.Unauthorized {
```

Comment:
A test would be nice, according to SC the scenario when `resp.Code() == codes.Unauthorized` is uncovered.